test/mysql_handler/tests/ssl_test.go

```diff
@@ -141,6 +141,50 @@ func TestSSL(t *testing.T) {
 				SSLRootCertType: Undefined,
 			},
 		},
+		{
+			TestDefinition: TestDefinition{
+				Description:         "server_tls, sslmode=verify-ca, sslrootcert=valid, sslkey=malformed, sslcert=malformed",
```

Comment:
Per our conversation today, let's change this description to be more human friendly / rspec like:

> It should error when the certificate or key is malformed

Same comment applies to other descriptions too.

Reply:
actually all the tests would need to be changed to this format. i'd opt for doing this in a follow up PR